### PR TITLE
SISRP-15737 Fix classname logging in ActiveAttr-using classes

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.berkeley.edu/browse/SISRP-15737

Finally tracked this occasionally crippling annoyance to its source...

### CODE REVIEW

File: app/models/user_specific_model.rb [hunk 1]
```diff
@@ -23,4 +23,10 @@ def directly_authenticated?
     @authentication_state.directly_authenticated?
   end
 
+  def logit
+    logger.warn 'whatever'
+    logger.warn "Here I am in UserSpecificModel code called from #{self.class} with ancestors #{self.class.ancestors}"
+    puts "method owner = #{self.method(:logger).owner}"
+  end
```
Comment (paulkerschen) [inline app/models/user_specific_model.rb]: @raydavis Was this method for diagnosis?

File: app/models/user_specific_model.rb [hunk 2]
```diff
@@ -23,4 +23,10 @@ def directly_authenticated?
     @authentication_state.directly_authenticated?
   end
 
+  def logit
+    logger.warn 'whatever'
```
Comment (christianvuerings) [inline app/models/user_specific_model.rb]: @raydavis this made me smile :)

File: app/models/user_specific_model.rb (hunk 1 above)
Comment (raydavis) [inline app/models/user_specific_model.rb]: Duh! Yes, to be amended.

**Comment by raydavis** [comment]
Embarrassing debris removed.

**Comment by paulkerschen** [comment]
Well ferreted, @raydavis.

**Pull request merged**